src/components/layout/footer.tsx

```
@@ -0,0 +1,74 @@
+import { createResource } from 'solid-js'
+
```

ESLint

[error] Missing semicolon.

(semi)